tasks/run.js

```diff
@@ -128,12 +128,14 @@ function runRouter (opts) {
 	(opts.localApps || [])
 		.concat({ name: normalizeName(packageJson.name, { version: false }), port: opts.localPort })
 		.forEach(function (localApp) {
+			const localAppEnvVar = localApp.name.toUpperCase().replace(/\-/, '_');
```

Comment:
`'email-article-api'.replace(/-/, '_')` yields `email_article-api`... you'll want the `/g` flag to replace all occurrences of the hyphen

Reply:
I had missed that! Thanks!